### PR TITLE
Updates and cosmetic changes for config doc

### PR DESCRIPTION
**Pull request by lamblin**

### CODE REVIEW

File: doc/library/config.txt [hunk 1]
```diff
@@ -392,23 +392,23 @@ import theano and print the config variable, as in:
 
     Bool value: either True or False
 
-    Default False
+    Default: False
```
Comment (nouiz) [inline doc/library/config.txt]: Put False in ````

**Comment by nouiz** [comment]
Other then my comment, it look good. Can you confirm that you generated the doc and it look good?

**Comment by lamblin** [comment]
Updated, as well as a couple of other places.
I did generate the doc locally, and it looks as intended.

**Comment by nouiz** [comment]
ok to merge when travis pass.

**Comment by lamblin** [comment]
Travis passed.

**Comment by lamblin** [comment]
I'll backport the changes to 0.8.X for the online doc. It is a bit tricky as there are new options.

**Comment by lamblin** [comment]
Done.

**Pull request merged**